src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,103 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed Status [/organizations/<organization_slug>/replays-viewed/]
+
+- Parameters
+
+  - project (optional, string)
+  - id (required, array[string]) - The id of the replay to retrieve. Can be specified multiple times. A maximum of 100 id's can be specified.
+
+### Browse Replays Viewed Status [GET]
+
+Returns a collection of replays and their viewed state. The user is targeted based on the authorization context. Replay-ids are specified in the URL parameters.
```

Comment:
```suggestion
Returns a collection of replay-ids and their viewed state. The user is targeted based on the authorization context. The returned ids are the same as those specified in the URL parameters -- excluding nonexistent ids.  
```

Reply:
We still want to include non-existent ids.  If the user hasn't viewed a replay yet a row won't exist and we should return false.

Reply:
Updated suggestion

Reply:
can we update the example response with this case?